source/dagger/gameplay/ping_pong/pingpong_playerinput.h

```diff
@@ -11,6 +11,8 @@ namespace ping_pong
     {
         EDaggerKeyboard up_key;
         EDaggerKeyboard down_key;
+        int inverted{1};
+        TimePoint inverted_time_point{};
```

Comment:
use Float32 timer instead of TimePoint, example is here:
https://github.com/UbiBelETF/dagger/pull/34/files

Reply:
Thanks for the advice , I'll change it asap

Reply:
Used Float32 instead of TimePoint as you mentioned , should be ok regarding that part now